Framework/Geometry/src/Objects/Object.cpp

```diff
@@ -1644,7 +1644,23 @@ void Object::calcBoundingBoxByGeometry() {
       maxZ = std::max(maxZ, vector.Z());
     }
   } break;
-
+  case 2: // HEXAHEDRON
```

Comment:
The comment is wrong, it needs to be changed. However, there is another pull request which uses a scoped enum instead of hard-coded integer values.